### DISCUSSION TITLE
update blocked or just me?

### DISCUSSION THREAD

**Discussion by unevoljitelj** [Joshua-Riek/ubuntu-rockchip#97] [General]
sudo apt update end with this but i would sware it was working at first install

Hit:1 http://ports.ubuntu.com/ubuntu-ports jammy InRelease
Hit:2 https://deb.nodesource.com/node_16.x jammy InRelease                                          
Get:3 http://ports.ubuntu.com/ubuntu-ports jammy-security InRelease [110 kB]                        
Get:4 http://ports.ubuntu.com/ubuntu-ports jammy-updates InRelease [119 kB]        
Ign:5 http://ports.ubuntu.com/ubuntu-ports jammy/main i386 Packages               
Ign:6 http://ports.ubuntu.com/ubuntu-ports jammy/restricted i386 Packages
Ign:7 http://ports.ubuntu.com/ubuntu-ports jammy/universe i386 Packages
Ign:8 http://ports.ubuntu.com/ubuntu-ports jammy/multiverse i386 Packages
Ign:5 http://ports.ubuntu.com/ubuntu-ports jammy/main i386 Packages
Hit:9 https://openhab.jfrog.io/artifactory/openhab-linuxpkg stable InRelease
Ign:6 http://ports.ubuntu.com/ubuntu-ports jammy/restricted i386 Packages
Ign:7 http://ports.ubuntu.com/ubuntu-ports jammy/universe i386 Packages
Ign:8 http://ports.ubuntu.com/ubuntu-ports jammy/multiverse i386 Packages
Ign:5 http://ports.ubuntu.com/ubuntu-ports jammy/main i386 Packages
Ign:6 http://ports.ubuntu.com/ubuntu-ports jammy/restricted i386 Packages
Ign:7 http://ports.ubuntu.com/ubuntu-ports jammy/universe i386 Packages
Ign:8 http://ports.ubuntu.com/ubuntu-ports jammy/multiverse i386 Packages
Ign:5 http://ports.ubuntu.com/ubuntu-ports jammy/main i386 Packages
Ign:6 http://ports.ubuntu.com/ubuntu-ports jammy/restricted i386 Packages
Ign:7 http://ports.ubuntu.com/ubuntu-ports jammy/universe i386 Packages
Ign:8 http://ports.ubuntu.com/ubuntu-ports jammy/multiverse i386 Packages
Ign:5 http://ports.ubuntu.com/ubuntu-ports jammy/main i386 Packages
Get:6 http://ports.ubuntu.com/ubuntu-ports jammy/restricted i386 Packages [174 kB]
Get:6 http://ports.ubuntu.com/ubuntu-ports jammy/restricted i386 Packages [174 kB]
Get:6 http://ports.ubuntu.com/ubuntu-ports jammy/restricted i386 Packages [174 kB]
Get:6 http://ports.ubuntu.com/ubuntu-ports jammy/restricted i386 Packages [174 kB]
Get:6 http://ports.ubuntu.com/ubuntu-ports jammy/restricted i386 Packages [174 kB]
Ign:6 http://ports.ubuntu.com/ubuntu-ports jammy/restricted i386 Packages
Ign:7 http://ports.ubuntu.com/ubuntu-ports jammy/universe i386 Packages
Ign:15 http://ports.ubuntu.com/ubuntu-ports jammy-security/main i386 Packages
Ign:16 http://ports.ubuntu.com/ubuntu-ports jammy-security/restricted i386 Packages
Ign:17 http://ports.ubuntu.com/ubuntu-ports jammy-security/universe i386 Packages
Ign:18 http://ports.ubuntu.com/ubuntu-ports jammy-security/multiverse i386 Packages
Ign:8 http://ports.ubuntu.com/ubuntu-ports jammy/multiverse i386 Packages
Err:5 http://ports.ubuntu.com/ubuntu-ports jammy/main i386 Packages
  404  Not Found [IP: 185.125.190.36 80]
Ign:6 http://ports.ubuntu.com/ubuntu-ports jammy/restricted i386 Packages
Ign:7 http://ports.ubuntu.com/ubuntu-ports jammy/universe i386 Packages             
Ign:15 http://ports.ubuntu.com/ubuntu-ports jammy-security/main i386 Packages
Ign:16 http://ports.ubuntu.com/ubuntu-ports jammy-security/restricted i386 Packages
Ign:17 http://ports.ubuntu.com/ubuntu-ports jammy-security/universe i386 Packages
Ign:18 http://ports.ubuntu.com/ubuntu-ports jammy-security/multiverse i386 Packages
Ign:8 http://ports.ubuntu.com/ubuntu-ports jammy/multiverse i386 Packages
Ign:15 http://ports.ubuntu.com/ubuntu-ports jammy-security/main i386 Packages
Ign:16 http://ports.ubuntu.com/ubuntu-ports jammy-security/restricted i386 Packages
Ign:17 http://ports.ubuntu.com/ubuntu-ports jammy-security/universe i386 Packages
Ign:18 http://ports.ubuntu.com/ubuntu-ports jammy-security/multiverse i386 Packages
Get:19 http://ports.ubuntu.com/ubuntu-ports jammy-updates/restricted i386 Packages [27,2 kB]
Get:19 http://ports.ubuntu.com/ubuntu-ports jammy-updates/restricted i386 Packages [27,2 kB]
Get:19 http://ports.ubuntu.com/ubuntu-ports jammy-updates/restricted i386 Packages [27,2 kB]
Get:19 http://ports.ubuntu.com/ubuntu-ports jammy-updates/restricted i386 Packages [27,2 kB]
Get:19 http://ports.ubuntu.com/ubuntu-ports jammy-updates/restricted i386 Packages [27,2 kB]
Ign:19 http://ports.ubuntu.com/ubuntu-ports jammy-updates/restricted i386 Packages
Ign:25 http://ports.ubuntu.com/ubuntu-ports jammy-updates/universe i386 Packages
Ign:26 http://ports.ubuntu.com/ubuntu-ports jammy-updates/multiverse i386 Packages
Ign:27 http://ports.ubuntu.com/ubuntu-ports jammy-updates/main i386 Packages
Ign:15 http://ports.ubuntu.com/ubuntu-ports jammy-security/main i386 Packages
Ign:16 http://ports.ubuntu.com/ubuntu-ports jammy-security/restricted i386 Packages
Ign:17 http://ports.ubuntu.com/ubuntu-ports jammy-security/universe i386 Packages
Ign:18 http://ports.ubuntu.com/ubuntu-ports jammy-security/multiverse i386 Packages
Ign:19 http://ports.ubuntu.com/ubuntu-ports jammy-updates/restricted i386 Packages
Ign:25 http://ports.ubuntu.com/ubuntu-ports jammy-updates/universe i386 Packages
Ign:26 http://ports.ubuntu.com/ubuntu-ports jammy-updates/multiverse i386 Packages
Ign:27 http://ports.ubuntu.com/ubuntu-ports jammy-updates/main i386 Packages
Ign:15 http://ports.ubuntu.com/ubuntu-ports jammy-security/main i386 Packages        
Ign:16 http://ports.ubuntu.com/ubuntu-ports jammy-security/restricted i386 Packages
Ign:17 http://ports.ubuntu.com/ubuntu-ports jammy-security/universe i386 Packages
Ign:18 http://ports.ubuntu.com/ubuntu-ports jammy-security/multiverse i386 Packages
Ign:19 http://ports.ubuntu.com/ubuntu-ports jammy-updates/restricted i386 Packages
Ign:25 http://ports.ubuntu.com/ubuntu-ports jammy-updates/universe i386 Packages
Ign:26 http://ports.ubuntu.com/ubuntu-ports jammy-updates/multiverse i386 Packages
Ign:27 http://ports.ubuntu.com/ubuntu-ports jammy-updates/main i386 Packages
Err:15 http://ports.ubuntu.com/ubuntu-ports jammy-security/main i386 Packages
  404  Not Found [IP: 185.125.190.36 80]
Ign:16 http://ports.ubuntu.com/ubuntu-ports jammy-security/restricted i386 Packages
Ign:17 http://ports.ubuntu.com/ubuntu-ports jammy-security/universe i386 Packages
Ign:18 http://ports.ubuntu.com/ubuntu-ports jammy-security/multiverse i386 Packages
Ign:19 http://ports.ubuntu.com/ubuntu-ports jammy-updates/restricted i386 Packages
Ign:25 http://ports.ubuntu.com/ubuntu-ports jammy-updates/universe i386 Packages
Ign:26 http://ports.ubuntu.com/ubuntu-ports jammy-updates/multiverse i386 Packages
Ign:27 http://ports.ubuntu.com/ubuntu-ports jammy-updates/main i386 Packages
Ign:19 http://ports.ubuntu.com/ubuntu-ports jammy-updates/restricted i386 Packages
Ign:25 http://ports.ubuntu.com/ubuntu-ports jammy-updates/universe i386 Packages
Ign:26 http://ports.ubuntu.com/ubuntu-ports jammy-updates/multiverse i386 Packages
Ign:27 http://ports.ubuntu.com/ubuntu-ports jammy-updates/main i386 Packages
Err:19 http://ports.ubuntu.com/ubuntu-ports jammy-updates/restricted i386 Packages
  404  Not Found [IP: 185.125.190.36 80]
Ign:25 http://ports.ubuntu.com/ubuntu-ports jammy-updates/universe i386 Packages
Ign:26 http://ports.ubuntu.com/ubuntu-ports jammy-updates/multiverse i386 Packages
Ign:27 http://ports.ubuntu.com/ubuntu-ports jammy-updates/main i386 Packages
Fetched 229 kB in 2s (116 kB/s)
Reading package lists... Done
E: Failed to fetch http://ports.ubuntu.com/ubuntu-ports/dists/jammy/main/binary-i386/Packages  404  Not Found [IP: 185.125.190.36 80]
E: Failed to fetch http://ports.ubuntu.com/ubuntu-ports/dists/jammy-security/main/binary-i386/Packages  404  Not Found [IP: 185.125.190.36 80]
E: Failed to fetch http://ports.ubuntu.com/ubuntu-ports/dists/jammy-updates/restricted/binary-i386/Packages  404  Not Found [IP: 185.125.190.36 80]
E: Some index files failed to download. They have been ignored, or old ones used instead.

**Comment by Joshua-Riek** [comment]
It's strange that you have i386 architecture added. Try a reboot or again and 30 minutes, sometimes the ubuntu servers can go down for a few minutes.

**Reply by unevoljitelj** [reply]
dunno how or why it was installed, but after googling how, i removed i386 arhitecture and now its updating fine with arm
